src/modules/pivotTable/PivotTableEngine.js

```diff
@@ -320,6 +323,19 @@ export class PivotTableEngine {
 
         const dataRow = this.data[row][column]
 
+        const ouId =
+            dataRow[
+                this.dimensionLookup.headerDimensions.findIndex(
+                    header => header.dimension === DIMENSION_ID_ORGUNIT
+                )
+            ]
+        const peId =
+            dataRow[
+                this.dimensionLookup.headerDimensions.findIndex(
+                    header => header.dimension === DIMENSION_ID_PERIOD
+                )
+            ]
+
```

Comment:
Is it possible that an `ou` or a `pe` header won't exist?  I think they should always exist, but if not this will return `-1` which will return the last column in the dataRow, and that's not what we want.

Reply:
Good point, not sure it's possible to not have `ou` and `pe` headers, but I can fix the code to avoid the wrong data returned.

Reply:
Fixed: a9c85c5065d2ae28c8f9dac0741b9dcf96296e26